src/rgw/rgw_cache.h

```diff
@@ -26,6 +37,140 @@ enum {
 #define CACHE_FLAG_MODIFY_XATTRS  0x08
 #define CACHE_FLAG_OBJV           0x10
 
+#define mydout(v) lsubdout(T::cct, rgw, v)
+
+/*DataCache*/
+struct DataCache;
+class L2CacheThreadPool;
+class HttpL2Request;
+
+struct ChunkDataInfo : public LRUObject {
```

Comment:
is the data cache sharing any code with the existing metadata cache stuff? if not, it really should be in separate files

Reply:
ack, moved to a separate file: src/rgw/rgw_d3n_datacache.h